src/main/java/thePackmaster/powers/darksoulspack/RemedyPower.java

```diff
@@ -0,0 +1,47 @@
+package thePackmaster.powers.darksoulspack;
+
+import com.evacipated.cardcrawl.mod.stslib.actions.tempHp.AddTemporaryHPAction;
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.OnReceivePowerPower;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class RemedyPower extends AbstractPackmasterPower implements OnReceivePowerPower {
+    public static final String POWER_ID = makeID("RemedyPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String DESCRIPTIONS[] = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    public RemedyPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, AbstractPower.PowerType.BUFF, false, owner, amount);
+        updateDescription();
+    }
+
+    @Override
+    public boolean onReceivePower(AbstractPower power, AbstractCreature target, AbstractCreature source) {
+        if (power.type == PowerType.DEBUFF && (amount > 0)){
```

Comment:
This doesn't work quite right with negative strength/dexterity/focus

Reply:
by this do you mean receiving negative str/dex/focus which wouldn't put you below 0?

Reply:
I mean any kind of negative str/dex/foc, be it from spire shield and spear or from somewhere else.

Negative strength is considered a debuff but the amount would be -2 so that would not be reduced by this power.

Reply:
just tested it, with the way it is coded currently it _does_ correctly reduce negative str/dex/foc. Amount in that snippet of code refers to the RemedyPower amount.

However thanks to that testing I found another issue, which is that instead of removing negative str/dex/foc, it actually gives you more. so i'll fix that.

Reply:
just realised maybe that is what you were trying to point out though 😅

Reply:
i think i've fixed this now :)